client/src/components/CourseReview.tsx

```diff
@@ -132,9 +217,34 @@ export const CourseReview = ({
             </Fragment>
           )}
         </p>
-        <h2 className='ml-auto mt-auto text-sm font-bold leading-none text-gray-700 dark:text-gray-200'>
-          {dateStr}
-        </h2>
+        <div className='flex items-center justify-end'>
+          <p className='mr-4 text-sm font-bold text-gray-700 dark:text-gray-200'>
+            {dateStr}
+          </p>
+          <div className='mb-0.5 flex items-center'>
+            <button className='flex h-8 w-8 items-center justify-center rounded-md text-gray-700 focus:outline-none dark:text-white'>
+              <AiFillLike
+                onClick={handleLike}
+                className={classNames(
+                  'h-4 w-4',
+                  kind === InteractionKind.Like ? 'fill-red-600' : ''
+                )}
+              />
+            </button>
+            <span className='text-sm font-bold text-gray-700 dark:text-white'>
+              {likes}
+            </span>
+          </div>
+          <button className='flex h-8 w-8 items-center justify-center rounded-md text-gray-700 focus:outline-none dark:text-white'>
+            <AiFillDislike
+              onClick={handleDislike}
+              className={classNames(
+                'h-4 w-4',
+                kind === InteractionKind.Dislike ? 'fill-red-600' : ''
+              )}
+            />
+          </button>
```

Comment:
Would probably be cleaner to extract this into its own component along with the interaction logic